doc/tutorials/acrn_on_qemu.rst

```diff
@@ -11,8 +11,8 @@ configuration.
 
 This setup was tested with the following configuration:
 
-- ACRN hypervisor: ``v2.6`` tag
-- ACRN kernel: ``v2.6`` tag
+- ACRN hypervisor: ``v2.7`` tag
+- ACRN kernel: ``v2.7`` tag
 - QEMU emulator version: 4.2.1
 - Service VM/User VM OS: Ubuntu 20.04
```

Comment:
This is not what the instructions below use, we seem to be using Ubuntu 18.04 instead:
```
--os-variant ubuntu18.04 \
...
--location 'http://archive.ubuntu.com/ubuntu/dists/bionic/main/installer-amd64/' \
```

Reply:
@gvancuts 
Ubuntu 20.04 is the native OS; not Service VM/User VM OS
acrn on qemu is verified for Service VM OS: 18.04

Any comments about following change?
From:
`Service VM/User VM OS: Ubuntu 20.04`
To:
`Ubuntu 20.04`

Thanks!

Reply:
My preference would be if we could consolidate onto a single version of Ubuntu, i.e. 20.04. Second best thing would be to keep these different version but specify them both, i.e.:
```
Host OS: Ubuntu 20.04
Service VM / User VM: Ubuntu 18.04
````